### PR TITLE
New command `npm run check`, automatically runs on prepublish.

### PR DESCRIPTION
**Pull request by koresar**
See #118

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
     "url": "git@github.com:stampit-org/stampit.git"
   },
   "dependencies": {
+    "babel-eslint": "^3.1.19",
```
Comment (ericelliott) [inline package.json]: Should this be in `devDependencies`?

Comment (koresar) [inline package.json]: Sometimes npm just can't do it right :)

File: package.json [hunk 2]
```diff
@@ -41,12 +42,16 @@
...
     "test": "npm run lint && babel-node test/index.js",
     "clean": "rimraf dist/* && mkdir dist || true",
     "uglify": "uglifyjs dist/stampit.js -m -c warnings=false -o dist/stampit.min.js",
     "lint": "eslint src && eslint test",
-    "build": "npm run clean && babel src --out-dir dist && npm run uglify"
+    "build": "npm run clean && babel src --out-dir dist && npm run uglify",
+    "deps": "npm run deps:missing && npm run deps:extra",
+    "deps:missing": "dependency-check package.json",
+    "deps:extra": "dependency-check package.json --extra --no-dev --ignore",
+    "check": "npm run deps && npm outdated --depth 0"
```
Comment (troutowicz) [inline package.json]: What is the purpose of running `npm outdated` here. It won't prevent publishing afaik, looks to be just some extra info for the developer.

Comment (koresar) [inline package.json]: Yeah. It won't stop from prepublishing, but will print a nice colourful table.

Also, we might use this `npm run check` separately, without publishing.

**Comment by troutowicz** [comment]
:+1:

**Pull request merged**